src/main/java/org/opentripplanner/standalone/config/RoutingRequestMapper.java

```diff
@@ -15,154 +16,303 @@ public class RoutingRequestMapper {
 
   private static final Logger LOG = LoggerFactory.getLogger(RoutingRequestMapper.class);
 
-  public static RoutingRequest mapRoutingRequest(NodeAdapter c) {
```

Comment:
I is probably easier to write a unit-test on this, rather than reviewing it and make sure everything is mapped.